src/ui/public/management/section.js

```diff
@@ -0,0 +1,68 @@
+import {assign} from 'lodash';
+import IndexedArray from 'ui/indexed_array';
+
+export default class ManagementSection {
+
+  /**
+   * @param {string} id
+   * @param {object} options
+   * @returns {ManagementSection}
+   */
+
+  constructor(id, options = {}) {
+    this.display = id;
+    this.id = id;
+    this.items = new IndexedArray({
+      index: ['id'],
+      order: ['order']
+    });
+
+    if (!options.url) {
+      options.url = '#/management/' + options.path || '';
+    }
+
+    assign(this, options);
+  }
+
+  /**
+   * Registers a section
```

Comment:
Could this read `Registers a sub-section`? I got really confused at first why `register` would be an instance method of ManagementSection.